suse_migration_services/units/setup_host_network.py

```diff
@@ -139,3 +142,18 @@ def log_network_details():
                 ).output
             )
         )
+
+
+def is_resolv_empty(resolv_conf_path):
+    with open(resolv_conf_path, 'r') as resolv:
+        # filter comment lines and empty lines
+        lines = filter(
+            lambda line: not line.startswith('#') and not line.startswith(os.linesep),
+            resolv.readlines()
+        )
+    for line in lines:
+        # check there is useful information in the remaining lines
+        if line.startswith('search') or line.startswith('nameserver'):
+            return False
+
+    return True
```

Comment:
Maybe we could write this a bit simpler ? like

```python
with open(resolv_conf_path, 'r') as resolv:
    for line in resolv:
        if line.startswith('search') or line.startswith('nameserver'):
            # looks like there is some effective resolver setup
            return False
return True
```

maybe it's also easier to read if the function is named "def have_host_resolv_setup" or similar

Thoughts ?

Reply:
Done.